src/openrct2-ui/interface/Widget.cpp

```diff
@@ -815,7 +815,7 @@ namespace OpenRCT2::Ui
         const auto& widget = w.widgets[widgetIndex];
 
         // Get the image
-        if (static_cast<int32_t>(widget.image.ToUInt32()) == SPR_NONE)
+        if (static_cast<int32_t>(widget.image.ToUInt32()) == kSpriteIdNull)
```

Comment:
The cast should be removed, i.e.:
```suggestion
        if (widget.image.ToUInt32() == kSpriteIdNull)
```
Similar in other such cases. Look at compiler output ('checks' on GitHub) for all similar cases.